pkg/api/devops/v2alpha1/pipelinerun_types.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2021.
```

Comment:
This copyright statement is different from other source files. Please keep them the same.

Reply:
> This copyright statement is different from other source files. Please keep them the same.

Okay, I will change it.